### PR TITLE
Simplify sim net

### PR DESCRIPTION
**Pull request by xiaoxiang781216**

### CODE REVIEW

**Comment by patacongo** [comment]
I will merge this when the conflicts are resolved.

**Comment by xiaoxiang781216** [comment]
@papatience the PR is rebased to the mainline now.

File: boards/sim/sim/sim/README.txt [hunk 1]
```diff
@@ -199,6 +199,16 @@ REVISIT: This may not long be an issue even with NSH because of the recent
...
 no longer freeze the simulation).
 
+Update: Please issue these commands to setup the reliable network on Ubuntu:
+
+  sudo apt-get -y install net-tools
+  sudo nuttx/tools/simbridge.sh eth0 on
+
+Here is some tips you may need:
+  1.Must launch the exectuable with the root permission
+  2.Have to use virtual machine if host is in corporation network
```
Comment (yamt) [inline boards/sim/sim/sim/README.txt]: why? just because it's better to isolate dev network?

Comment (xiaoxiang781216) [reply boards/sim/sim/sim/README.txt]: In my case, the coperation network enable some adavanced link layer authentication which NuttX doesn't implement yet and then fail to login the network.

**Pull request merged**